private-identity-audit/src/verifier.rs

```diff
@@ -143,54 +148,49 @@ mod tests {
 
         // Test original anonymity param.
         let (_, committed_uids, _) = VerifierSetGenerator::generate_committed_set_and_challenge(
-            gen_random_uuids(input_len, &mut rng)
-                .into_iter()
-                .map(uuid_to_scalar)
-                .collect(),
+            PrivateUids(
```

Comment:
Dedup this code into a function. It is used in the following lines: 151, 167, and 183